packages/emitter-framework/src/typescript/components/type-alias-declaration.tsx

```diff
@@ -1,23 +1,29 @@
 import { refkey as getRefkey } from "@alloy-js/core";
 import * as ts from "@alloy-js/typescript";
-import { Scalar } from "@typespec/compiler";
+import { Type } from "@typespec/compiler";
 import { $ } from "@typespec/compiler/experimental/typekit";
 import { reportDiagnostic } from "../../lib.js";
 import { TypeExpression } from "./type-expression.jsx";
 
 export interface TypedAliasDeclarationProps extends Omit<ts.TypeDeclarationProps, "name"> {
-  type: Scalar;
+  type: Type;
   name?: string;
 }
 
 export type TypeAliasDeclarationProps = TypedAliasDeclarationProps | ts.TypeDeclarationProps;
 
+/**
+ * Create a TypeScript type alias declaration. Pass the `type` alias to emit the
+ * type as the provided TypeSpec type.
```

Comment:
```suggestion
 * type as the provided TypeSpec type.
 * @example
 * ```ts
 * type Foo = ReferencedType;
 * ```
 
```

Reply:
I feel like @example should show example, possibly in addition to example output?

Reply:
I think this should read "Pass the type **prop**", I'll fix.